src/test/resources/bandits/bandits-parameters-1.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
@giorgiomartini0 This is the part you'll want to focus on to make sure I correctly understood the takeaways from our sync

Reply:
Thank you! Did a light pass for now. Some of the questions I left above are also here in the code (and I noticed a few discrepancies for categorical coefficients).